### DISCUSSION TITLE
reading an excel file in production

### DISCUSSION THREAD

**Discussion by rohitmi0023** [vercel/next.js#21465] [Help]
I have an excel file `Words.xlsx` in public directory. Initially, during development mode, `/public/Words.xlsx` worked fine but it failed in production mode saying that it can't read file.Then, I read [this](https://nextjs.org/docs/basic-features/static-file-serving) and changed path to`/Words.xlsx` but I am still receiving the same error in my function logs.
```js
[Error: ENOENT: no such file or directory, open '/Words.xlsx'] {
  errno: -2,
  code: 'ENOENT',
  syscall: 'open',
  path: '/Words.xlsx'
}
ENOENT: no such file or directory, open '/Words.xlsx'
```  
How do I solve this? Do I have to upload file in AWS and read it from there?

**Comment by yunushkh9** [comment]
Please first update your  node.js file
